docs/storage.md

```diff
@@ -179,6 +189,38 @@ Now you can create the SR where `server` is your mon ip.
 * You may specify `serverport` option if you are using any other port than 6789.
 :::
 
+### Ceph iSCSI gateway
+
+This is at this moment the only way to connect to Ceph with no modifications of dom0, it's possible to create multiple Ceph iSCSI gateways following this: https://docs.ceph.com/docs/master/rbd/iscsi-target-cli/
+
+For some reason the chap authentication between gwcli and Xen doesn't seem to be working, so it's recommended to disable it (in case you use no authentication a dedicated network for storage should be used to ensure some security).
+
+IMPORTANT: User had many weird glitches with iSCSI connection via ceph gateway in lab setup (3 gateways and 3 paths on each host) after several days of using it. So please keep in mind that this setup is experimental and unstable. This would have to be retested on recent XCP-ng.
+
+### Ceph RBD
```

Comment:
Shouldn't this section explain how to get `rbd` on XCP-ng? And if it requires external packages, make sure to link to https://xcp-ng.org/docs/additionalpackages.html too.

Please also add a foreword stating that this is to be considered experimental.

Reply:
A warning is added for both.